XRDXRFutils/gaussnewton.py

```diff
@@ -86,21 +85,13 @@ def gamma(self):
         return self.w(self.g)
 
 
-    # @staticmethod
-    # def u(x):
-    #     return GaussNewton.w(100 * x) / 100
-
-    # @staticmethod
-    # def der_u(x):
-    #     return GaussNewton.der_w(100 * x)
-
     @staticmethod
```

Comment:
```
    @staticmethod
    def w(x):
        return 0.5 * (sqrt(x**2 + 1) + x)
```

I am more inclined to use the above. I worry that it can oscillate otherwise.

Reply:
As a first choice, I used that `w()` to define `sigma2`, but I obtained wild oscillations during fit.
So, I tried 2 options:
```
u(x) = w(100x) / 100
u(x) = x^2
```
The first one rescales `w` so that it is closer to a piece-wise linear function. The idea is to have the range of usual `tau` on the part of the function with slope near to 1.
Both options worked quite well. I chose the second because I find it conceptually simpler.